pipeline_games/load_games.py

```diff
@@ -0,0 +1,289 @@
+"""Script for loading to database"""
+from os import environ
+from dotenv import load_dotenv
+import pandas as pd
+from psycopg2 import connect, DatabaseError, sql
+from psycopg2.extras import RealDictCursor, execute_batch
+
+
+def get_db_connection(config) -> connect:
+    """Connect to the database with game data"""
+    try:
+        return connect(
+            user=config['DATABASE_USERNAME'],
+            password=config['DATABASE_PASSWORD'],
+            host=config['DATABASE_IP'],
+            port=config['DATABASE_PORT'],
+            database=config['DATABASE_NAME'])
+    except ValueError:
+        return "Error connecting to database."
+
+
+def execute_batch_columns(conn, data: pd.DataFrame, table: str, column: str, page_size=100) -> None:
+    """batch execution of adding specified data to the database"""
+    tuples = list(zip(data.unique()))
+    cols = column
+    query = """INSERT INTO %s(%s) VALUES(%%s)""" % (table, cols)
+    with conn.cursor(cursor_factory=RealDictCursor) as cur:
+        try:
+            execute_batch(cur, query, tuples, page_size)
+            conn.commit()
+            cur.close()
+            print("execute_batch() done")
+        except (Exception, DatabaseError) as error:
+            print(f"Error: {error}")
+            conn.rollback()
+            cur.close()
+
+
+def execute_batch_columns_for_genres(conn, data: pd.DataFrame, table: str, page_size=100) -> None:
+    """batch execution of adding genres into the database"""
+    tuples = [tuple(x) for x in data.to_numpy()]
+    cols = ','.join(list(data.columns))
+    query = "INSERT INTO %s(%s) VALUES(%%s,%%s) " % (table, cols)
```

Comment:
I like that. Instead of manually adding column names assuming all will be matching - this works perfectly